decentraland/components/presentational/testHelpers.js

```diff
@@ -0,0 +1,47 @@
+import AssetTypes from "@constants/AssetTypes";
+const { ESTATE, PARCEL } = AssetTypes;
+
+export const estateForSale = {
+  id: "1",
+  priceMana: 0,
+  priceUSD: 0,
+  asset: {
+    type: ESTATE,
+    id: "123",
+    name: "Sample Estate",
+    img: "https://api.decentraland.org/v1/estates/1/map.png",
+  },
+};
+
+export const parcelForSale = {
+  id: "1",
+  priceMana: 0,
+  priceUSD: 0,
+  asset: {
```

Comment:
Maybe use `parcel` in the `parcelForSale` as the `asset` value?

Reply:
Done

Reply:
Thanks! Resolving